includes/Data/Brands.php

```diff
@@ -14,6 +14,62 @@ final class Brands {
 	public static function get_brands() {
 
 		return array(
+			'newfold'       => array(
+				'name'                        => 'your web host',
```

Comment:
Should we be passing `your web host` to the `__` helper function, since it is not a proper noun?